pkg/secret/secret.go

```diff
@@ -34,7 +34,8 @@ Please delete the component and recreate it using 'odo create'`, componentName)
 			return secrets[0].Name, nil
 		}
 		return "", fmt.Errorf(`Multiple secrets exist for component %s. 
-Please select one of the following ports: '%s'`, componentName, strings.Join(availablePorts(secrets), ","))
+Please select one of the following ports: '%s' 
+by supplying the --port option and rerun the command`, componentName, strings.Join(availablePorts(secrets), ","))
```

Comment:
as you are using back ticks `` ` ``, the new line is actually not ignored and the `by supplying ...` prints in the next line.
Might wanna consider
 ```
"stuff " +
"next line" 
```

Reply:
Hm... The message I am getting in the terminal  is the following:

```
None of the secrets that exist for component backend match port 9324. 
Please select one of the following ports: '8080' 
by supplying the --port option and rerun the command
```

 which looks correct. What am I missing here?

Reply:
the wrapping of the message should be taken care by the terminal, try to resize the terminal once. This is what it looks like to me
![image](https://user-images.githubusercontent.com/6551988/51023288-9501eb80-15ac-11e9-9374-d8358b3ecb64.png)

Reply:
so we shouldn't add a new line in the message but new line in the code so its more readable.